src/ui/forms/forms.tsx

```
@@ -0,0 +1,58 @@
+  values,
+  handleChange,
+  onSubmit,
+  submitLabel
+}: FormProps<T>) {
+  return (
+    <form onSubmit={onSubmit}>
+      <fieldset>
+        <legend>{legend}</legend>
+          {fields.map((field) => {
+            return (
+              <div className={styles.formRow}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)